Sources/StreamChat/Database/DatabaseContainer.swift

```diff
@@ -224,8 +224,14 @@ class DatabaseContainer: NSPersistentContainer {
 
     /// Removes all data from the local storage.
     func removeAllData(completion: ((Error?) -> Void)? = nil) {
+        /// Cleanup the current user cache for all manage object contexts.
+        allContext.forEach { context in
+            context.perform {
```

Comment:
Should we block here with `performAndWait` like we do with writableContext just below? Or should we chain performs (writableContext below + allContext here) and then calling the completion handler?

Reply:
@laevandus It should be ok to not wait here since it is a quite fast operation, and the other one which takes more time we will wait for it 👍